src/frontend/components/Felleskomponenter/TekstBlock.tsx

```diff
@@ -6,7 +6,7 @@ import { useApp } from '../../context/AppContext';
 import { LocaleRecordBlock } from '../../typer/common';
 import { EFlettefeltverdi, ESanitySteg } from '../../typer/sanity/sanity';
 
-const TekstBlock: React.FC<{ block: LocaleRecordBlock; barnetsNavn?: string }> = ({
+const TekstBlock: React.FC<{ block: LocaleRecordBlock | undefined; barnetsNavn?: string }> = ({
```

Comment:
Er ikke dette det samme?
```suggestion
const TekstBlock: React.FC<{ block?: LocaleRecordBlock; barnetsNavn?: string }> = ({
```